docs/src/main/asciidoc/redis.adoc

```diff
@@ -592,6 +592,14 @@ Quarkus supports a feature called DevServices that allows you to create various
 What that means practically, is that if you have docker running and have not configured `quarkus.redis.hosts`,
 Quarkus will automatically start a Redis container when running tests or dev-mode, and automatically configure the connection.
 
+Available properties to customize the Redis DevServices.
+[source,properties]
+----
+quarkus.redis.devservices.image-name=redis:6.0-alpine
+quarkus.redis.devservices.port=6377
+quarkus.redis.devservices.enabled=true (true by default)
```

Comment:
```suggestion
quarkus.redis.devservices.enabled=true
```

Reply:
No need for this as we have the generated doc which tells us what default values are